src/wazuh_modules/wmodules.c

```diff
@@ -308,15 +308,31 @@ cJSON *getModulesConfig(void) {
 int modulesSync(char* args) {
     int ret = -1;
     wmodule *cur_module = NULL;
-    for (cur_module = wmodules; cur_module; cur_module = cur_module->next) {
-        if (strstr(args, cur_module->context->name)) {
-            ret = 0;
-            if (strstr(args, "dbsync") && cur_module->context->sync != NULL) {
-                ret = cur_module->context->sync(args);
+    size_t retry = 0;
+
+    do {
+        if (retry > 0) {
+            usleep(retry * WM_MAX_WAIT);
+            mdebug1("At modulesSync(): WModules is not ready. Retry %d", retry);
```

Comment:
The IDE shows a warning about the type of `retry`: it is a **size_t** variable but we are printing `%d` 

![Captura de pantalla de 2023-05-02 17-04-17](https://user-images.githubusercontent.com/65046601/235773960-f8d3903a-d860-4dc3-a96c-49cc9d72b13a.png)